docs/sources/reference/run.md

```diff
@@ -134,6 +134,11 @@ While not strictly a means of identifying a container, you can specify a version
 image you'd like to run the container with by adding `image[:tag]` to the command. For
 example, `docker run ubuntu:14.04`.
 
+### Image[@digest]
+
+Images using the v2 or later image format may be referred to using their
```

Comment:
@moxiegirl The second sentence bears a connotation that if I update the image content the former digest will no longer be accessible. Also I think that the digest value is always predictable after all (so that different registries arrive at the same digest for the same input regardless of any consensus protocol being involved).

Essentially I'd change the *tone* from negative, describing what digests are not, to positive, in the vein of:
" Digests are identifiers computed from both manifest metadata and image layer content, hence they provide a unique absolute identifier for the given input content & metadata."